guides/03_API_Overview.md

```diff
@@ -48,6 +48,34 @@ When obtaining devices to configure, the browser may initially decline to provid
 
 You can override the behavior of the device-label trigger by calling meetingSession.audioVideo.[setDeviceLabelTrigger(trigger)](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#setdevicelabeltrigger).
 
+#### Implement a view-only/observer/spectator experience
+
+We recommend that you use `deviceLabelsTrigger` to control device permissions.
```

Comment:
Could we introduce the problem in the starting paragraph, "To enable a ...."

Reply:
could we also say "the device permission prompts for audio and video"

Reply:
Also clarify that by doing so tthey are not impacted in their ability to view content or listen in on audio in their meeting experience

Reply:
How about:
"To enable a view-only mode, you need to control the device permission. We recommend that you implement it based on 'deviceLabelsTrigger`."

Reply:
> Also clarify that by doing so tthey are not impacted in their ability to view content or listen in on audio in their meeting experience
Sure, I will add it to the React Component how-to as well. It is good to mention this.

Reply:
Updated.